design/sketch.webidl

```diff
@@ -574,16 +574,15 @@ enum GPUPrimitiveTopology {
 
 dictionary GPURenderPipelineDescriptor : GPUPipelineDescriptorBase {
     GPUPipelineStageDescriptor vertexStage;
-    GPUPipelineStageDescriptor fragmentStage;
+    GPUPipelineStageDescriptor? fragmentStage = null;
 
-    GPUPrimitiveTopology primitiveTopology;
+    GPUPrimitiveTopology primitiveTopology = "triangle-list";
```

Comment:
We resolved today to make this required.

Reply:
Hmm, any reason why? Non-triangle-list topologies are just bad ideas on most modern GPUs.

Reply:
AFAIK all topologies exposed by WebGPU are find on modern hardware. Triangle strip is often better than triangle list because it can keep vertices in the cache better.

Reply:
triangle-strip (and primitive reset) are a pain for the input assembler since you can't just chunk things out in parallel, you have to serially parse the stream. indexed buffers are a much better approach to cache optimization than stripification. even mobile GPU vendors do not recommend strips anymore.

Reply:
@magcius that's an interesting statement, and it makes sense. Do you have any links backing it up?

Reply:
Surprisingly enough, I could not find any public statement about it in the GPU performance guides I looked at. It's one of those "tribal knowledge" gamedev things, and a careful reading of the GCN paper would get you to the same conclusion.

Reply:
Haven't heard this opinion before, despite coming from "gamedev tribe" myself. If it's not documented, let's not base the decisions on it.

Reply:
100% agreed. I suspected there would be public documentation about it (I was told this by a mobile GPU vendor) but there does not appear to be any.